### PR TITLE
NW6 | Fikret Ellek | React-Module-Project | Week-2 | orderType

### PR DESCRIPTION
**Pull request by fikretellek**
## Learners, PR Template

Self checklist

- [ ] I have committed my files one by one, on purpose, and for a reason
- [ ] I have titled my PR with COHORT_NAME | FIRST_NAME LAST_NAME | REPO_NAME | WEEK 
- [ ] I have tested my changes
- [ ] My changes follow the [style guide](https://syllabus.codeyourfuture.io/guides/code-style-guide/)
- [ ] My changes meet the [requirements](./README.md) of this task

## Changelist

Briefly explain your PR.

## Questions

Ask any questions you have for your reviewer.

### CODE REVIEW

File: src/components/Order/Order.jsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import RestaurantButton from "../RestaurantButton/RestaurantButton";
+import React, { useState } from "react";
+import "./Order.scss";
+
+export default function Order({ orderType }) {
+  const [orders, setOrders] = useState(0);
```
Comment (AltomHussain) [inline src/components/Order/Order.jsx:6]: We always create the SetState var to be camel case. So your `setorder` should be camel case. Can you change that please?

File: src/components/Order/Order.jsx [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  const handleAddOrder = () => {
+    setOrders(orders + 1);
+  };
+
+  return (
+    <li className="restaurant__item">
+      {orderType}: {orders}
+      <RestaurantButton handleAddOrder={handleAddOrder} />
```
Comment (AltomHussain) [inline src/components/Order/Order.jsx:14]: Nice work

File: src/components/Restaurant/Restaurant.jsx [hunk 3]
```diff
@@ -1,21 +1,13 @@
...
       <h3 className="restaurant__heading">Restaurant Orders</h3>
       <ul className="restaurant__list">
-        <li className="restaurant__item">
-          Orders: {orders}
-          <button className="button restaurant__button" onClick={handleAddOrder}>
-            Add
-          </button>
-        </li>
+        <Order orderType="Pizzas" />
+        <Order orderType="Salads" />
```
Comment (AltomHussain) [inline src/components/Restaurant/Restaurant.jsx:9]: Perfect work!

**Review by AltomHussain — COMMENTED** [review]
Good work!

**Review by RbAvci — APPROVED** [review]

File: src/components/Restaurant/Restaurant.test.jsx [hunk 4]
```diff
@@ -15,19 +15,23 @@ describe("Restaurant", () => {
...
+    const addButtons = screen.getAllByTestId("addButton");
+    addButtons.map((addButton) => fireEvent.click(addButton));
 
-    const updatedPizzaText = screen.getByText("Orders: 1");
+    const updatedText1 = screen.getByText("Pizzas: 1");
+    const updatedText2 = screen.getByText("Salads: 1");
+    const updatedText3 = screen.getByText("Chocolate cake: 1");
 
-    expect(updatedPizzaText).toBeInTheDocument();
+    expect(updatedText1, updatedText2, updatedText3).toBeInTheDocument();
```
Comment (Ara225) [inline src/components/Restaurant/Restaurant.test.jsx:35]: All looking good, tests are really good as well

**Pull request merged**